plugins/logger.rst

```diff
@@ -1,4 +1,27 @@
 Logger Plugin
 =============
 
-TODO
+Logger plugin logs request, response and exception coming from an `Http\Client\HttpClient` or
```

Comment:
The `LoggerPlugin` converts requests, responses and exceptions to strings and logs them with a PSR3_ compliant logger::